example/index.html

```diff
@@ -8,8 +8,8 @@
     <link rel="stylesheet" href="style.min.css">
     <link rel="stylesheet" href="../dist/yoast-seo.min.css">
 
-    <script type="text/javascript" src="example-scraper.min.js"></script>
-    <script type="text/javascript" src="../dist/yoast-seo.min.js"></script>
+    <script type="text/javascript" src="example-scraper.js"></script>
+    <script type="text/javascript" src="../dist/yoast-seo.js"></script>
 
```

Comment:
Is the use of the non-minified for development purposes only, or intended for the live version?